lib/spawn-browser.ts

```diff
@@ -72,14 +79,23 @@ class BrowserProcess implements IBrowserProcess {
   private lastError: Error;
   private hasExited: boolean = false;
 
-  constructor(executablePath: string, args: string[]) {
+  constructor(
+    executablePath: string,
+    args: string[],
+    stdio: "pipe" | "ignore" | "inherit" | null = "inherit",
+  ) {
+    // the types aren't current for this
+    // disabling buffer used to be maxBuffer: null
+    // now it is buffer: false
     const child = execa(executablePath, args, {
-      // disable buffer, pipe to stdout
-      maxBuffer: null as any,
-    });
-    child.stdout.pipe(process.stdout);
-    child.stderr.pipe(process.stderr);
-    child.on("error", (err: Error) => (this.lastError = err));
+      // disable buffer, pipe or drain
+      buffer: false,
+      stdio,
+    } as any);
```

Comment:
Can we not type this?

Reply:
it is typed, but the types on DefinitelyTyped are stale. It used to be maxBuffer: null was how you disabled buffer but it changed in 1.0